src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -308,6 +308,27 @@ char* GetCpuModel(void* log)
     return textResult;
 }
 
+unsigned int GetNumberOfCpuCores(void* log)
+{
+    const char* osCpuCoresCommand = "lscpu | grep \"CPU(s):\"";

```

Comment:
is `lscpu` widely available in our support-matrix?

Reply:
Of course, yes.